### PR TITLE
[NEW]  Added a link to contributing.md

### LINKED ISSUE #12845: Link CONTRIBUTING.md in "How to contribute"
Would be better if [CONTRIBUTING.md](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md) would be linked directly from README's [How to Contribute](https://github.com/RocketChat/Rocket.Chat#how-to-contribute) section, because at first I thought there wasn't any [CONTRIBUTING.md](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md) in this project. Would send a PR if this idea is appreciated. :)

### PR DESCRIPTION
**Pull request by sanketsingh24**
Closes #12845

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ If you want to help, send an email to support at rocket.chat to be invited to th
 
 ## How to Contribute
 
-Already a JavaScript developer? Familiar with Meteor? [Pick an issue](https://github.com/RocketChat/Rocket.Chat/labels/contrib%3A%20easy), push a PR and instantly become a member of Rocket.Chat's international contributors community.
+Already a JavaScript developer? Familiar with Meteor? [Pick an issue](https://github.com/RocketChat/Rocket.Chat/labels/contrib%3A%20easy), push a PR and instantly become a member of Rocket.Chat's international contributors community. For more information, check out our [Contributing Guide](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md).
```
Comment (geekgonecrazy) [inline README.md]: might be a good idea to make this link relative.  Something like (.github/CONTRIBUTING.md)

Comment (sanketsingh24) [reply README.md]: Oh yea, forgot about that :sweat_smile:

**Comment by MartinSchoeler** [comment]
@sanketsingh24 @tassoevan @geekgonecrazy How about linking to our [contributing](https://rocket.chat/docs/contributing/) Docs too?

**Comment by tassoevan** [comment]
@MartinSchoeler Yes, but we need to check if there is no conflicting information between the texts.

**Comment by sanketsingh24** [comment]
Well, didn't even knew something like that even existed :sweat_smile: will add that too.

**Pull request merged**